CMakeLists.txt

```diff
@@ -54,6 +54,15 @@ if(NOT ENABLE_DEVICES)
 endif()
 message(STATUS "Enable Devices: ${ENABLE_DEVICES}")
 
+# if we are using Cuda, also add a "CudaUVM" device to use that memory space
+# in the test suite. Also enable a Cabana UVM option so the tests get triggered.
+foreach(_device ${ENABLE_DEVICES})
+  if(_device STREQUAL Cuda)
+    list(APPEND CABANA_SUPPORTED_DEVICES CudaUVM)
+    set(Cabana_ENABLE_CudaUVM ON)
```

Comment:
Where are these used?

Reply:
This get used in the unit test `CMakeLists.txt` - any time you see `if(Cabana_ENABLE_${_device})` it will then include a UVM device. The way we have our device settings means that we need two testing device types when CUDA is enabled - one with UVM and one without. This line adds the extra UVM device.

Reply:
OK